ftw/referencewidget/profiles/default_plone5/registry.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<registry>
+    <records interface="ftw.referencewidget.interfaces.IReferenceSettings" />
+
+    <records prefix="plone.resources/referencewidget_css" interface="Products.CMFPlone.interfaces.IResourceRegistry">
+        <value key="css">
+            <element>++resource++ftw.referencewidget/refwidget.css</element>
+        </value>
+    </records>
+
+    <records prefix="plone.resources/handlebar_js" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
```

Comment:
The name `handlebar_js` has conflict potential.
I suggest prefixing all the resources:
```suggestion
    <records prefix="plone.resources/ftw.referencewidget-handlebar_js" interface='Products.CMFPlone.interfaces.IResourceRegistry'>
```